modules/javafx.web/src/main/native/Source/ThirdParty/libxml/UPDATING.txt

```diff
@@ -0,0 +1,43 @@
+Updating libxml in OpenJFX:
+
+1. Download respective libxml source tarball from here: http://xmlsoft.org/sources
```

Comment:
As an alternative, the source can be downloaded by cloning:
https://gitlab.gnome.org/GNOME/libxml2.git
They tag each release. Maybe it isn't worth noting here.

Reply:
Added the git url also as an alternate source in the steps.